libs/libcommon/src/libcommon/storage_client.py

```diff
@@ -92,11 +86,26 @@ def get_unsigned_url(self, path: str) -> str:
         return url
 
     def sign_url_if_available(self, url: str) -> str:
-        if self.cloudfront:
-            url = self.cloudfront.sign_url(url=url)
+        if self.url_signer:
+            url = self.url_signer.sign_url(url=url)
             logging.debug(f"signed url: {url}")
         return url
 
+    def sign_urls_in_obj(self, obj: Any) -> Any:
```

Comment:
I think It is better to handle all this logic in the signer instead to avoid giving extra responsibilities to a Storage Client class.

Reply:
yes, it makes sense, I'll change this.

Reply:
done